pom.xml

```diff
@@ -179,9 +179,9 @@
     <version.org.glassfish.jaxb>2.3.2</version.org.glassfish.jaxb>
     <version.org.gwtbootstrap3>1.0.1</version.org.gwtbootstrap3>
     <version.org.gwtbootstrap3-extras>1.0.2</version.org.gwtbootstrap3-extras>
-    <version.org.hibernate>5.3.7.Final</version.org.hibernate>
+    <version.org.hibernate>5.3.14.Final</version.org.hibernate>
```

Comment:
@mareknovotny Very very minor question: I checked `WildFly18` BOM and the hibernate version is `5.3.13`. Is this intentional? Probably there is no problem to use `5.4.13`, just want to report it.
https://github.com/wildfly/wildfly/blob/26689e876b7c5d271e3d023539fa57c52bad503d/pom.xml#L334

Reply:
@yesamer this is more inline with what EAP 7.3.0 will bring

Reply:
@mareknovotny Thank you!

Reply:
@yesamer @mareknovotny  hiberante 5.4.14 version will rise problems in sql scripting and validation. better be done that bump by eng than prod team.

Reply:
@elguardian it is upgrade to 5.3.14 not 5.4.14 ;) so the upgrade is on micro version only.

Reply:
@mareknovotny I was answering @yesamer bout bumping to 5.14... there is one problem.

Reply:
@elguardian Yes, I wrongly wrote `5.4.14` version in my first comment, which was a type ( I meant `5.3.14`). My apologies for the typo and thank you for your clarification.